javav2/usecases/pam_source_files/src/main/java/com/example/photo/endpoints/RestoreEndpoint.java

```diff
@@ -24,21 +24,31 @@ public RestoreEndpoint(DynamoDBService dynamoDBService, S3Service s3Service, Sns
     }
 
     public Job restore(String notify, List<String> tags) {
-        String topicArn = this.snsService.createNotificationTopic(notify);
-
         Set<String> images = tags.stream().parallel().flatMap(this::imagesByTag).collect(Collectors.toSet());
         String manifest = makeManifest(PhotoApplicationResources.STORAGE_BUCKET, images);
         String manifestArn = this.s3Service.putManifest(manifest);
-        String jobId = this.s3Service.startRestore(manifestArn, tags);
 
-        Job job = new Job(jobId, topicArn);
+        // These three values are required for createJob() to work: etag. accountID, roleARN!
+        String eTag = this.s3Service.getETag(manifestArn, PhotoApplicationResources.WORKING_BUCKET);
+
+        // This needs to be pulled in from CDK!
+        String accountID = "924034042979";  // REPLACE WITH ACCOUNT ID
```

Comment:
This comes from the lambda ARN, see https://www.radishlogic.com/aws/lambda/how-to-get-the-aws-account-id-in-lambda-node-js/ and https://stackoverflow.com/questions/49163151/programmatically-get-account-id-from-lambda-context-arn

Reply:
This is not needed in new app flow